lua/rustaceanvim/config/internal.lua

```diff
@@ -283,7 +283,7 @@ local RustaceanDefaultConfig = {
     end,
     ---@type string[] | fun():(string[]|fun(dispatchers: vim.lsp.rpc.Dispatchers): vim.lsp.rpc.PublicClient)
     cmd = function()
-      return { 'rust-analyzer', '--log-file', RustaceanConfig.server.logfile }
+      return { vim.fn.exepath('rust-analyzer'), '--log-file', RustaceanConfig.server.logfile }
```

Comment:
issue: `vim.fn.exepath` can return an empty string, with would break the error message in the health check. We should fall back to `'rust-analyzer'` in this case.